src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -341,10 +345,16 @@ public sealed override ValueTask OnDefinitionFoundAsync(DefinitionItem definitio
                     return null;
                 }
 
+                var (guid, projectName, projectFlavor) = GetGuidAndProjectInfo(document);
+
                 return DocumentSpanEntry.TryCreate(
                     this,
                     definitionBucket,
-                    documentSpan,
+                    guid,
+                    projectName,
+                    projectFlavor,
+                    document.FilePath,
+                    documentSpan.SourceSpan,
```

Comment:
i didn't want to padd 'documentSpan' along (in case it got accidentally captured in the future), so i explicitly broke out the data we computed from it and passed it along here insetad.